tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -279,3 +279,525 @@ namespace EnergyPlus {
 	}
 
 }
+
+TEST( DXCoilsTest, TestMultiSpeedDefrostCOP )
+{
+	// Test that the COP calculation is correct when the defrost is on. #4973
+	
+	using CurveManager::Quadratic;
+	using CurveManager::BiQuadratic;
+	using CurveManager::NumCurves;
+	using EnergyPlus::DataEnvironment::OutHumRat;
+	using EnergyPlus::DataEnvironment::OutDryBulbTemp;
+	using EnergyPlus::DataEnvironment::OutBaroPress;
+	using DXCoils::CalcMultiSpeedDXCoilHeating;
+	int DXCoilNum;
+	
+	// Set up heating coil and curves.
+	
+	InitializePsychRoutines();
```

Comment:
@nmerket The HVACFixture takes care of this as well so this is no longer needed here.